ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -40,7 +40,6 @@ build_root:
     name: release
     namespace: openshift
     tag: golang-1.10
-test_binary_build_commands: chmod uga+w /etc/passwd
```

Comment:
btw, we basically need this `chmod` in the `src` image from what I can tell since we're using `openshift_installer_src` and we're in `src` anyway